build_script.py

```diff
@@ -307,13 +308,19 @@ def main(args=sys.argv[1:]):
     build_parser.add_argument(
         "--build-dir",
         help="Path to the output build directory. If not specified, a "
-             "temporary directory is used",
+             "temporary directory is used.",
         default=tempfile.mkdtemp())
     build_parser.add_argument(
         "--foundation-build-dir",
         help="Path to swift-corelibs-foundation build products, which "
              "the built XCTest.so will be linked against.",
         required=strategy.requires_foundation_build_dir())
+    build_parser.add_argument(
+        "--foundation-install-prefix",
+        help="Path to the installation location for swift-corelibs-foundation "
+             "build products; CoreFoundation dependencies are expected to be "
+             "found under $foundation_build_dir/$foundation_install_prefix.",
```

Comment:
I think it'd be nice to include in the description what the default value is. For example, the `--lit` argument does this: `"'%(default)s' by default."`.